invoices/invoiceregistry.go

```diff
@@ -199,6 +220,27 @@ func (i *InvoiceRegistry) invoiceEventNotifier() {
 				i.singleNotificationClients[e.id] = e
 			}
 
+		case event := <-i.htlcAutoReleaseChan:
```

Comment:
Tried out launching the goroutine in the main registry event loop. What do you think? I am a bit hesitant to optimize this already now with a priority queue.

Reply:
Trying out prio queue now

Reply:
Pushed